app/scripts/components/common/map/map-component.tsx

```diff
@@ -70,8 +77,10 @@ export default function MapComponent({
       initialViewState={initialViewState}
       mapStyle={style as any}
       onMove={onMove}
+      onLoad={onMapLoad}
       projection={mapboxProjection}
       maxBounds={maxMapBounds}
+      interactive={interactive}
```

Comment:
What is this flag for? (Also I couldn't find this attribute from the doc: https://visgl.github.io/react-map-gl/docs/api-reference/map am I missing anything?)

Reply:
Yeah, we need them for the ScrollyMap. The 'interactive' prop is coming from `MapBox`

> Aside from the props listed below, the Map component supports all parameters of the Map class constructor ([Mapbox](https://docs.mapbox.com/mapbox-gl-js/api/map/) | [Maplibre](https://maplibre.org/maplibre-gl-js-docs/api/map/)).

https://visgl.github.io/react-map-gl/docs/api-reference/map#properties

Reply:
Ah right, thanks for the explanation!